src/components/landing/PastAnnouncements.tsx

```diff
@@ -0,0 +1,77 @@
+"use client";
+
+import CalendarPlainIcon from "@/icons/CalendarPlainIcon";
+import { Announcement } from "@/models/Announcement";
+import { NO_ANNOUNCEMENTS_MESSAGE } from "@/utils/constants";
+import {
+  filterPastAnnouncements,
+  sortAnnouncementsByStartDate,
+} from "@/utils/utils";
+import ActiveCell from "./AnnouncementCell";
+import { useState, useEffect } from "react";
+import ButtonSecondary1 from "../shared/ButtonSecondary1";
+
+interface Props {
+  announcements: Announcement[];
+}
+
+export default function PastAnnouncements({ announcements }: Props) {
+  const pastAnnouncements = sortAnnouncementsByStartDate(
+    filterPastAnnouncements(announcements)
+  );
+
+  const [annLimit, setAnnLimit] = useState(3);
+
+  useEffect(() => {
+    const updateLimit = () => {
+      const screenWidth = window.innerWidth;
+      if (screenWidth <= 768) {
+        setAnnLimit(1);
+      } else {
+        setAnnLimit(3);
+      }
+    };
+
+    updateLimit();
+    window.addEventListener("resize", updateLimit);
+
+    return () => window.removeEventListener("resize", updateLimit);
+  }, []);
```

Comment:
Is this logic needed? For the default behavior (mobile), simply use `announcements[0]` and apply `md:hidden`. Then, set the limit to 3 for tablet and desktop and use `max-md:hidden` for that.